<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrtWindow</class>
 <widget class="QDialog" name="CrtWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>288</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>0</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Steckmodule</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="spacing">
            <number>2</number>
           </property>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <widget class="WidgetFileBrowse" name="FileBrowser" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>8</pointsize>
                </font>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item>
                <widget class="QPushButton" name="InsertCRT">
                 <property name="minimumSize">
                  <size>
                   <width>100</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>100</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Modul aktivieren</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="RemoveCRT">
                 <property name="minimumSize">
                  <size>
                   <width>110</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>110</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>Modul deaktivieren</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="ViewChipData">
                 <property name="minimumSize">
                  <size>
                   <width>22</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>22</width>
                   <height>23</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>8</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string notr="true">+</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>2</number>
             </property>
             <item>
              <widget class="TreeWidgetMod" name="CRTInfo">
               <property name="minimumSize">
                <size>
                 <width>239</width>
                 <height>106</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>239</width>
                 <height>106</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>7</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 100);</string>
               </property>
               <property name="indentation">
                <number>0</number>
               </property>
               <property name="itemsExpandable">
                <bool>true</bool>
               </property>
               <column>
                <property name="text">
                 <string>Variable</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>7</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Inhalt</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>7</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
               </column>
               <item>
                <property name="text">
                 <string>Name</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Version</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Hardware Typ</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>EXPROM Leitung</string>
                </property>
                <property name="text">
                 <string/>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GAME Leitung</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Chip Anzahl</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="TreeWidgetMod" name="ChipList">
               <property name="minimumSize">
                <size>
                 <width>239</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>240</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Arial</family>
                 <pointsize>7</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">color: rgb(0, 0, 100);</string>
               </property>
               <property name="indentation">
                <number>0</number>
               </property>
               <column>
                <property name="text">
                 <string>Nr.</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>7</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Chip Typ</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>7</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Bank</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>7</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Adresse</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>7</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
               </column>
               <column>
                <property name="text">
                 <string>Größe</string>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>7</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
               </column>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="TreeWidgetMod" name="ChipData">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>457</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>0</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>7</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 127);</string>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <column>
            <property name="text">
             <string notr="true">1</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QStackedWidget" name="MoreCRTPage">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>0</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="PageEasyFlash">
          <widget class="QWidget" name="widget_2" native="true">
           <property name="geometry">
            <rect>
             <x>7</x>
             <y>7</y>
             <width>160</width>
             <height>50</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(:/grafik/easy_flash_logo.png);</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="PageFC"/>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="NewEasyFlashCRT">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>Leeres EasyFlash erstellen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetFileBrowse</class>
   <extends>QWidget</extends>
   <header>widget_file_browse.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TreeWidgetMod</class>
   <extends>QTreeWidget</extends>
   <header>tree_widget_mod.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
